client/my-sites/site-monitoring/index.ts

```diff
@@ -7,7 +7,7 @@ export default function () {
 	page( '/site-monitoring', siteSelection, sites, makeLayout, clientRender );
 
 	page(
-		'/site-monitoring/:siteId',
+		'/site-monitoring/:siteId/:tab(php|web|null)?',
```

Comment:
I don't think this is intentional. The `?` makes the `tab` param optional which should be all we need I think. With the `|null` in the regex means that the `/site-monitoring/:siteId/null` is a valid route and it will render the web logs.

```suggestion
		'/site-monitoring/:siteId/:tab(php|web)?',
```